argilla-server/src/argilla_server/contexts/distribution.py

```diff
@@ -51,6 +53,11 @@ async def update_record_status(search_engine: SearchEngine, record_id: UUID) ->
 
         await db.commit()
 
+        await notify_record_event(db, RecordEvent.updated, record)
+
+        if record.is_completed():
+            await notify_record_event(db, RecordEvent.completed, record)
```

Comment:
Are you sure we want to have a `record.updated` event and a `record.completed` event with the same info? I believe we should not notify the `record.updated` if we are gonna notify the `record completed` right?

But not 100% sure.